docs/getting-started/hive.md

```diff
@@ -0,0 +1,33 @@
+---
+title: "Installing Apache Hive"
+slug: /getting-started/hive
+license: "This software is licensed under the Apache License version 2."
+---
+
+To install Apache Hive and Hadoop on Google Cloud Platform manually,
+follow [Apache Hive](https://cwiki.apache.org/confluence/display/Hive/) and
+[Hadoop](https://hadoop.apache.org/docs/stable/hadoop-project-dist/hadoop-common/SingleCluster.html).
+
+Installing and configuring Hive can be a little complex.
+If you don't already have Hive set up and running, you can use the Docker container
+Datastrato provides to get Gravitino up and running.
```

Comment:
Breaking this sentence, hide an error. "that Datastrato provides" would be better I think.

Reply:
Will fix.